### DISCUSSION TITLE
To assign a value superDatasetIndexShardsFactor didn't take effect

### DISCUSSION THREAD

**Discussion by liyuntian** [apache/skywalking#11425] [General]
Assign superDatasetIndexShardsFactor = 5 and superDatasetIndexReplicasNumber = 0, then index need to be modified , adjust superDatasetIndexShardsFactor superDatasetIndexShardsFactor = 3, after restart skywalking, configuration take effect in application.yml But the super size dataset record index is calculated according to superDatasetIndexShardsFactor = 5,NOT superDatasetIndexShardsFactor = 3. why?   skywalking version:9.2.0

**Comment by wu-sheng** [comment]
Changing this would not effect the created templates. That is never recommended for a product environment. If you need to adjust the template about anything, change that on your Elasticsearch server side.

**Reply by liyuntian** [reply]
![image](https://github.com/apache/skywalking/assets/10043467/75405200-7ad2-493e-b52f-6080df7778a8)
I want to change that,but I get all my template is empty

**Reply by wu-sheng** [reply]
If you want to change the template, read and learn from Elastic document. That is not our reponsibility.
